item-api/src/main/java/com/example/itemapi/global/config/CacheType.java

```diff
@@ -0,0 +1,19 @@
+package com.example.itemapi.global.config;
+
+import lombok.Getter;
+
+@Getter
+public enum CacheType {
+    ITEM_INFO("itemInfo", 24 * 60 * 60, 10000)
+    ;
+
+    private String cacheName;
```

Comment:
```suggestion
    private final String cacheName;
```

기본적으로 private 변수는 final 을 선언해주면 좋다고 생각해요 (개취)